MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -951,24 +952,21 @@ private void processPrisonerCapture(List<TestUnit> unitsToProcess) {
                 if (null != en.getCrew()) {
                     currentHits = en.getCrew().getHits();
                 }
-                int newHits = (int) Math.max(0, currentHits - existingHits);
-                casualties = (int) Math.ceil(Compute.getFullCrewSize(en) * (newHits/6.0));
+                int newHits = Math.max(0, currentHits - existingHits);
+                casualties = (int) Math.ceil(Compute.getFullCrewSize(en) * (newHits / 6.0));
             }
-            for(Person p : crew) {
-                // Give them a UUID. We won't actually use this for the campaign, but to
-                //identify them in the prisonerStatus hash
-                UUID id = p.getId();
-                if (null == id) {
-                    id = UUID.randomUUID();
-                    p.setId(id);
-                }
-                PrisonerStatus status = new PrisonerStatus(p.getFullName(), u.getEntity().getDisplayName(), p);
+            for (Person p : crew) {
+                // We need to assign them a UUID now to place them into the hash
```

Comment:
You probably still need to say which hash.

Reply:
Added clarification to the comment in question.